coordinator/src/position/models.rs

```diff
@@ -27,8 +27,16 @@ pub struct NewPosition {
 #[derive(PartialEq, Debug)]
 pub enum PositionState {
     Open,
-    Closing,
-    Closed { pnl: i64 },
+    /// The position is in the process of being closed
+    ///
+    /// Once the position is being close the closing price is known.
+    /// For backwards compatibility we make the closing price optional.
+    Closing {
+        closing_price: f32,
```

Comment:
But it is _not_ optional here.

Reply:
Thanks for catching this one! I eventually opted for just defaulting it to `0` when mapping it into the business logic model instead of making it optional there. It is optional in the db model :)